src/server/host/Endpoints/Administration/Events/UpdateEventEndpoint.cs

```diff
@@ -57,7 +62,7 @@ public override async Task HandleAsync(UpdateEventRequest req, CancellationToken
             return;
         }
 
-        if (!eventInfo.Staged)
+        if (req.Commit != null && !eventInfo.Staged)
```

Comment:
I think the error should only appear if you try to set `Commit` to `false` when the even is already committed.
If you send `true` and it is already committed, that is fine as nothing really changes.

```suggestion
        if (req.Commit == false && !eventInfo.Staged)
```